apps/aecore/lib/aecore/chain/worker.ex

```diff
@@ -28,7 +30,9 @@ defmodule Aecore.Chain.Worker do
   def init(_) do
     genesis_block_hash = BlockValidation.block_header_hash(Block.genesis_block().header)
     genesis_block_map = %{genesis_block_hash => Block.genesis_block()}
-    genesis_chain_state = ChainState.calculate_and_validate_chain_state!(Block.genesis_block().txs, %{}, 0)
+    genesis_chain_state =
+      ChainState.calculate_and_validate_chain_state!(Block.genesis_block().txs, %{accounts: %{}}, 0)
```

Comment:
maybe make a `ChainState.empty()` funtion to return a base structure `%{accounts: %{}}` to reduce possible errors by changing the structure